### DISCUSSION TITLE
Seatbelt switch events

### DISCUSSION THREAD

**Discussion by bluedoors** [flybywiresim/aircraft#4116] [Help] [answered]
Running Spad.NeXt to view SIMCONNECT events when changing the position of the seltbelt switch it doesn't appear to fire the "CABIN SEATBELTS ALERT SWITCH". I'm trying to find a way to have my external hardware panel operate the seatbelt switch (and the Chimes for what its worth). Is there a SIMCONNECT event for this, or would i be better using the FS SDK to change that value (and would FBW pick it up)

**Comment by mojoair** [comment]
There is a SIMCONNECT event for this, however it does not appear to be working yet on the A32NX. 

It does work on default A320 and so it seems like an A32NX issue. Maybe can report it to the team?

FYI the event ID is : 66719   CABIN_SEATBELTS_ALERT_SWITCH_TOGGLE

**Comment by hotshotp** [comment, marked as answer]
look at #4040  - it will fix your issue when it is merged
